tests/ff_module-inline-edit.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var React = require('react');
+require('./lib/utils').bootstrapBrowser();
+var TestUtils = require('react-addons-test-utils');
+var expect = require('chai').expect;
+
+var InlineEdit = require('../blocks/core/ff_module/ff_module-inline-edit/ff_module-inline-edit');
+
+describe('InlineEdit', function() {
+	it('should render an inline-edit', function() {
+		var element = React.createElement(InlineEdit);
+		var component = TestUtils.renderIntoDocument(element);
+		expect(component).to.exist;
+	});
+
+	var url = '[url]';
```

Comment:
Can you add the vars before the `describe` block? It's generally considered best practise to declare vars together at the top.